data/battle_scripts_1.s

```diff
@@ -417,7 +417,77 @@ gBattleScriptsForMoveEffects::
 	.4byte BattleScript_EffectVictoryDance            @ EFFECT_VICTORY_DANCE
 	.4byte BattleScript_EffectTeatime                 @ EFFECT_TEATIME
 	.4byte BattleScript_EffectAttackUpUserAlly        @ EFFECT_ATTACK_UP_USER_ALLY
-	.4byte BattleScript_EffectShellTrap				  @ EFFECT_SHELL_TRAP
+	.4byte BattleScript_EffectShellTrap               @ EFFECT_SHELL_TRAP
+	.4byte BattleScript_EffectCeaselessEdge           @ EFFECT_CEASELESS_EDGE
+	.4byte BattleScript_EffectDireClaw                @ EFFECT_DIRE_CLAW
+	.4byte BattleScript_EffectStoneAxe                @ EFFECT_STONE_AXE
+	.4byte BattleScript_EffectBarbBarrage             @ EFFECT_BARB_BARRAGE
+
+BattleScript_EffectStoneAxe::
```

Comment:
Perhaps it would be a good idea to merge BattleScript_EffectCeaselessEdge and BattleScript_EffectStoneAxe into one battlescript? We could use the argument field in the battle moves to set which hazard to set up.

Reply:
Done. Will push when I have every correction-suggestion ready :eyes: